FirebaseAdmin/FirebaseAdmin/Auth/Utils.cs

```diff
@@ -45,12 +45,27 @@ internal static string ResolveIdToolkitHost(string projectId, IdToolkitVersion v
                 throw new ArgumentException("Must provide a project ID to resolve");
             }
 
-            var emulatorHostEnvVar = EnvironmentVariable.FirebaseAuthEmulatorHost;
-            var useEmulatorHost = !string.IsNullOrWhiteSpace(emulatorHostEnvVar);
             var versionAsString = version.ToString().ToLower();
-            return useEmulatorHost
-                ? string.Format(IdToolkitEmulatorUrl, emulatorHostEnvVar, versionAsString, projectId)
-                : string.Format(IdToolkitUrl, versionAsString, projectId);
+
+            var emulatorHostEnvVar = EnvironmentVariable.FirebaseAuthEmulatorHost;
+            if (!string.IsNullOrWhiteSpace(emulatorHostEnvVar))
+            {
+                return string.Format(IdToolkitEmulatorUrl, emulatorHostEnvVar, versionAsString, projectId);
+            }
+
+            return string.Format(IdToolkitUrl, versionAsString, projectId);
+        }
+
+        internal static class EnvironmentVariable
+        {
+            internal const string FirebaseAuthEmulatorHostName = "FIREBASE_AUTH_EMULATOR_HOST";
+
+            /// <summary>
+            /// Gets environment variable for connecting to the Firebase Authentication Emulator.
+            /// The variable is expected to be in the form &lt;host&gt;:&lt;port&gt;, e.g. localhost:9099.
+            /// </summary>
+            internal static string FirebaseAuthEmulatorHost
```

Comment:
You can put this directly in the Utils class, and call it `EmulatorHostEnvironmentVar`. Callers can refer to it as `Utils.EmulatorHostEnvironmentVar`, which is quite clear and in terms of readability.

Reply:
I've removed it entirely, since there only was one callsite for it.